### ISSUE TITLE
Task: The colour contrast on the details component is below the minimum threshold.

### ISSUE THREAD

**Issue by DeanWillicombe82** [ministryofjustice/data-platform#163]
### Task Description

The colour contrast on the details component is below the minimum threshold. We need to amend the colour to amend to ensure we meet WCAG criteria.

Search for data assets - Find MOJ data - GOV.UK  

https://find-moj-data.service.justice.gov.uk/search?subject_area=urn:li:tag:Courts%20and%20tribunals

### Acceptance Criteria

Colour Contrast for details component is within WCAG acceptable criteria

### Steps to Address

UX Designer to put forward recommendations to change to colour. Create a sub-task using implementation issue type.
Developer to implement and test on implementation ticket. 

### Useful Contacts

matt.proctor-leake@justice.gov.uk

**Comment by AdamDMOJ** [comment]
This has been designed and annotated [here](https://www.figma.com/design/MZij9V8VJUJmJ7LTAPFwCs/FindMOJdata--2026-?m=auto&t=I490uXouxmaTOQot-6) and is ready for review.

It has been brought in line with [1.4.3 Contrast (Minimum) (Level AA)](https://webaim.org/standards/wcag/checklist#sc1.4.3) from 3.86 (Fail) to 5.47 (Pass AA)

Please do let me know if more information is needed.
